x/fundraising/spec/02_state.md

```diff
@@ -145,10 +145,10 @@ type FixedPriceAuction struct {
 type BatchAuction struct {
     *BaseAuction
     
-	MinBidPrice           sdk.Dec           // the minimum bid price that bidders must provide
-	MatchedPrice          sdk.Dec           // the matched price of the auction (a.k.a., winning price)
+	MinBidPrice			sdk.Dec	// the minimum bid price that bidders must provide
+	MatchedPrice		sdk.Dec	// the matched price of the auction (a.k.a., winning price)
```

Comment:
Can you check the indentation? 
https://github.com/jeonghoyeo7/fundraising/blob/128-fix-cli-placeBid/x/fundraising/spec/02_state.md

Reply:
Checked and modified.